test/house_test.rb

```diff
@@ -0,0 +1,70 @@
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_it_has_a_price
+    assert_equal "$400000", @house.price
+  end
+
+  def test_it_has_an_address
+    assert_equal "123 sugar lane", @house.address
+  end
+
+  def test_it_initializes_with_empty_rooms_array
+    assert_equal [], @house.rooms
+  end
+
+  def test_it_can_add_rooms_and_store_in_rooms
```

Comment:
Nice job testing this method thoroughly here. We actually don't need the test on line 31 or 33, because that's a command method `.add_room` that we use to setup our query method `.rooms` that our command method is changing. A better setup would be: 
```
    room_1 = Room.new(:bedroom, 10, '13')
    room_2 = Room.new(:bedroom, 11, '15')
    assert_equal [],  @house.rooms
    @house.add_room(room_1)
    assert_equal [@room_1], @house.rooms
    @house.add_room(room_2)
    assert_equal [@room_1, @room_2], @house.rooms
```